DQMOffline/Configuration/python/DQMOffline_cff.py

```diff
@@ -153,7 +154,9 @@
 
 #Post sequences are automatically placed in the EndPath by ConfigBuilder if PAT is run.
 #miniAOD DQM sequences need to access the filter results.
-PostDQMOfflineMiniAOD = cms.Sequence(miniAODDQMSequence*jetMETDQMOfflineSourceMiniAOD*tracksDQMMiniAOD*topPhysicsminiAOD)
+from DQMOffline.Muon.miniAOD_cff import * 
+
+PostDQMOfflineMiniAOD = cms.Sequence(miniAODDQMSequence*jetMETDQMOfflineSourceMiniAOD*tracksDQMMiniAOD*topPhysicsminiAOD*muonMonitors_miniAOD*MuonMiniAOD)
```

Comment:
muonMonitors_miniAOD is a copy of muonMonitors with less functionality therefore I think  it could in the same place as muonMonitors, that is a normal Path.  We'll move it to a normal Path.